### PR TITLE
Adding option to define a backend for `.Values.ingress.extraHosts`

### LINKED ISSUE #213: Ingress `extraHosts` does not allow different backend
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Summary

For the ingress backend, I would like to be able to specify `serviceName` and `servicePort`.

### Motivation

As an SRE, I don't want someone to try to ClickOps an upgrade in the UI.  We should have the ability to block certain admin paths.

### Describe the solution

I will put in a PR this afternoon.

### PR DESCRIPTION
**Pull request by unacceptable**
Closes #213

### CODE REVIEW

File: charts/yourls/values.yaml [hunk 1]
```diff
@@ -477,7 +477,13 @@ ingress:
...
   ## extraHosts:
   ##   - name: yourls.local
-  ##     path: /
+  ##     path: /foo
+  ##     pathType: Prefix # optional
+  ##     backend:         # optional
+  ##       service:
+  ##         name: response-foo
+  ##         port:
+  ##           name: use-annotation
```
Comment (unacceptable) [inline charts/yourls/values.yaml:486]: This backend example is specific to the aws load balancer ingress controller: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/ingress/annotations/#actions

**Comment by unacceptable** [comment]
@LeoColomb are you a project maintainer?  If so could I get a review on this?

**Comment by LeoColomb** [comment]
@unacceptable Yes, I'm the maintainer. I take vacations, sometimes.

**Comment by unacceptable** [comment]
@LeoColomb Excellent! I will look forward to your input when you're back from vacation. Enjoy your time off.

**Comment by LeoColomb** [comment]
Thanks for your PR @unacceptable.
That being said, I'm not sure to understand the need.
As far as I understand the need on #213, everything is already covered by current templates.

Reusing your example, the following should be fine:
```yaml
extraHosts:
  - name: yourls.local
    path: /foo
    serviceName: response-foo
    servicePort: use-annotation
```

**Pull request closed without merging**